docs/_snippets/contribution-process.mdx

```diff
@@ -0,0 +1,18 @@
+<Steps>
+    <Step title="Fork the Mage GitHub repo">
+    Visit [GitHub](https://github.com/mage-ai/mage-ai) and fork the Mage repo to your machine.
```

Comment:
might just be semantics, but you fork the repo and then clone it to your machine